src/snowflake/snowpark/async_job.py

```diff
@@ -147,15 +147,15 @@ class AsyncJob:
             >>> from snowflake.snowpark.functions import col
             >>> query_id = session.sql("select 1 as A, 2 as B, 3 as C").collect_nowait().query_id
             >>> async_job = session.create_async_job(query_id)
-            >>> async_job.query
+            >>> async_job.query # doctest: +SKIP
```

Comment:
I have only seen this line failing, have the others all been flaky?

Reply:
No, but in the cases where this line fails, the following lines do not get a chance to run (and do not have a chance to be surfaced as flaky lines). 

I reasoned that if the query test isn't working, it is likely that the result would be flaky as well.

Reply:
I don't think the following lines are necessarily dependent on the first line. I prefer only skipping the ones that are shown to be flaky. cc: @sfc-gh-jdu

Reply:
Sure, skipping only the two flaky lines.